game/scripts/vscripts/TutorialGraph/Steps.ts

```diff
@@ -115,6 +115,48 @@ export const setCameraTarget = (target: CBaseEntity | undefined) => {
     })
 }
 
+/**
+ * Moves the camera to a unit, with lerp
+ * @param target Unit to move the camera to.
+ * @param lerp Speed at which the camera moves
+ */
+export const moveCameraToUnit = (target: CBaseEntity, lerp: number) => {
+    let playerIds = findAllPlayersID();
+
+    playerIds.forEach(playerId => {
+        let player = PlayerResource.GetPlayer(playerId);
+
+        if (player) {
+            CustomGameEventManager.Send_ServerToPlayer(player, "move_camera", {
+                unitTargetEntIndex: target.GetEntityIndex(),
+                lerp: lerp
+            })
+        }
+    })
+}
+
+/**
+ * Moves the camera to a position, with lerp
+ * @param position Point to move the camera to.
+ * @param lerp Speed at which the camera moves
+ */
+export const moveCameraToPosition = (position: Vector, lerp: number) => {
+    let playerIds = findAllPlayersID();
```

Comment:
I think you can just use `findRealPlayerID()` instead to get the player's ID. There's only one real player.